code/segmentedtickerplant/stplog.q

```diff
@@ -290,14 +290,24 @@ init:{[dbname]
     // add the info to the meta table
     .stpm.updmeta[multilog][`open;logtabs;.z.p+.eodtime.dailyadj];
     ]
+
+  // set log handles to null in sctp if not in create mode
+  if[.sctp.chainedtp and not .sctp.loggingmode=`create; 
+    `..loghandles set .stplg.t!(count .stplg.t)#(::);
+    // only need to log errors if sctp is creating its own logs
+    `.stplg.badmsg set {[x;y;z]};
```

Comment:
With a namespace variable you can just use equals, ie.
`.stplg.badmsg:{[x;y;z]};`